collector/consumer/exporter/otelexporter/defaultadapter/adapter.go

```diff
@@ -0,0 +1,37 @@
+package defaultadapter
+
+import (
+	"github.com/Kindling-project/kindling/collector/model"
+	"go.opentelemetry.io/otel/attribute"
+)
+
+type Adapter interface {
+	Adapt(group *model.GaugeGroup) ([]*AdaptedResult, error)
+
+	// Transform Only Aggregator Value should use the Transform
+	Transform(group *model.GaugeGroup) (*model.AttributeMap, error)
```

Comment:
What difference is it between `Adapt` and `Transform`? Could you describe it as comments?